File: templates/java/es/text/code/invertLine.java
import stanford.karel.*;
public class InverConos extends Karel {
   /** iniciar la ejecución aquí */
   public void run() {
      while(frenteDespejado()) {
         invertirCono();
         moverse();
      }
      /* para prevenir un error de poste */
      invertirCono();
   }

   /*  recoge un cono si uno está presente coloca un cono
    * contrario */
   public void invertirCono() {
      /* una declaración if / else */
      if(conosPresentes()) {
         recogerCono();
      } else {
         ponerCono();
      }
   }
}

File: templates/python/es/text/code/invertLine.py
from karel.stanfordkarel import *

# el inicio del programa
def main():
   while frente_despejado(): 
      invertir_cono()
      moverse()
   # para prevenir un error en el poste de la cerca
   invertir_cono()

# recoge un cono si hay uno presente
# pone un cono contrario
def invertir_cono():
   # una declaración if / else
   if conos_presente():
      recoger_cono()
   else:
      poner_cono()